rs/embedders/src/serialized_module.rs

```diff
@@ -177,12 +189,12 @@ impl OnDiskSerializedModule {
             data_segments: serialized_module.data_segments,
             wasm_metadata: serialized_module.wasm_metadata,
         };
-        let mut bytes_file = File::create(bytes_path)
+        let mut bytes_file = File::create_new(bytes_path)
```

Comment:
Not very important I guess, but wouldn't it help if we handled the error more explicitly to be able to differentiate between cases where `create_new` fails because the file exists vs. other failure cases?

Reply:
Yeah makes sense.